gutenberg.php

```diff
@@ -136,18 +136,7 @@ function gutenberg_init( $return, $post ) {
 		return false;
 	}
 
-	$post_type        = $post->post_type;
-	$post_type_object = get_post_type_object( $post_type );
-
-	if ( 'attachment' === $post_type ) {
```

Comment:
Noting that we're no longer checking this in `gutenberg_can_edit_post_type`, and while I don't know the original reason why we were explicitly targeting `attachment`, my understanding is this should still be effective since `post_type_supports( 'attachment', 'editor' ) === false`.

Reply:
Yah, that's why I dropped it, it was an unnecessary extra check. :-)